src/components/SidebarLink.tsx

```diff
@@ -31,11 +31,17 @@ export const SidebarLink = forwardRef<unknown, SidebarLinkProps>(
     }
 
     const active = getActive(pathname, slug);
-
     const isActive = cx({
       active,
     });
 
+    const label = item.label.replaceAll(/[_-]/g, ' ');
+    const shouldBeLowerCase =
+      slug.startsWith('docs/forc/commands/') ||
+      slug.startsWith('docs/forc/plugins/') ||
+      slug.startsWith('docs/indexer/forc-index/') ||
+      slug.startsWith('docs/indexer/forc-postgres/');
```

Comment:
Also this?

We should find a way to abstract these things to avoid many checkings using string. It makes really hard to maintain.